nautobot_golden_config/forms.py

```diff
@@ -430,9 +420,14 @@ class Meta:
 
 
 class RemediationSettingFilterForm(NautobotFilterForm):
-    """Filter Form for Line Replacement."""
+    """Filter Form for Remediation Settings."""
 
     model = models.RemediationSetting
+    q = forms.CharField(required=False, label="Search")
+    platform = utilities_forms.DynamicModelMultipleChoiceField(queryset=Platform.objects.all(), required=False)
+    remediation_type = forms.ChoiceField(
+        choices=add_blank_choice(RemediationTypeChoice), required=True, label="Remediation Type"
```

Comment:
```suggestion
        choices=add_blank_choice(RemediationTypeChoice), required=False, label="Remediation Type"
```

Double check me on this, but I don't think any filter form should have required fields.

Reply:
had already fixed this one locally. Still broken when filters are actually run.